src/UIAutomation/UiaOperationAbstraction/UiaTypeAbstractionImpl.g.cpp

```diff
@@ -7008,7 +7008,10 @@
             auto element = std::get<AutomationRemoteElement>(m_member).GetParentElement();
             if (cacheRequest)
             {
-                element.PopulateCache(*cacheRequest);
+                delegator->If(element.IsNull().BoolNot(), [&]()

```

Comment:
Could we abstract this into a helper?

Reply:
I copy/paste this from what code generator generates, I prefer to keep this as it is.

Reply:
Fair